sql/wsrep_schema.cc

```diff
@@ -1117,7 +1117,10 @@ static int remove_fragment(THD*                  thd,
                  seqno.get(),
                  error);
     }
-    ret= error;
+    else
+    {
+      ret= error;
```

Comment:
This looks fine but can we tighten the condition here i.e. ignore not found only if transaction is bf killed ?

Reply:
Unfortunately, this is not possible... or probably more complicated than it looks. The `thd` that removes fragments is not necessarily the same `thd` that executes the transaction. So we can't just lookup the state of the `thd`.